src/backend.js

```diff
@@ -106,6 +97,17 @@ var _ = Mavo.Backend = $.Class({
 			call.searchParams.set("timestamp", Date.now());
 		}
 
+		if ($.type(req.data) === "object") {
+			if (req.method == "GET") {
+				Object.keys(req.data).map(p => call.searchParams.set(p, req.data[p]));
```

Comment:
Nitpick: `map()` implies that you are mapping one array to another, and plan to use the result. If you're just looping over an array, use `forEach` which communicates your intent better.

Also, in this case you don’t need either, just a for...in loop would do:

```js
for (var p in req.data) {
	call.searchParams.set(p, req.data[p]);
}
```

(make sure to check this works, I haven't)